README.md

```diff
@@ -115,7 +115,9 @@ This will create the `audits` and `audit_deltas` tables that will store each obj
 Applying the `AuditableBehavior` to a model is essentially the same as applying any other CakePHP behavior. The behavior does offer a few configuration options:
 
 - ``ignore`` An array of property names to be ignored when records are created in the deltas table.
+- ``ignoreDeep`` Boolean `true`/`false` to ignore the above array of property names in **HABTM** and **join table** when records are created in deltas table. Defaults to false. If a property name exists in `ignore` as well as `deepLog`, it will still be ignored.
```

Comment:
Yeah, I know. It is not used unless explicitly set to true. Just thought it would matter if you are desperate to limit the size of logged HABTM data, like I do. It seems I just overthought this, I could easily use ``deepLog`` to control it anyway (except that it will not ignore any properties of join table, which was my target here).